packages/boiler-task-selenium/test/get-test-files-spec.js

```diff
@@ -0,0 +1,120 @@
+// Libraries
+import Immutable from 'immutable';
+import {expect} from 'chai';
+// Helpers
+import makeMockConfig from './config/make-mock-config';
+import getTestFiles, {makeSpecsGlob} from '../src/get-test-files';
+
+
+const baseSpecDir = 'test/e2e/wdio';
+const runnerOptions = { specsDir: baseSpecDir };
+describe(`#makeSpecsGlob`, () => {
+  it(`should return a generic spec glob if both 'desktop' and 'mobile' are specified`, () => {
+    const config = makeMockConfig({desktop: true, mobile: true});
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/**/*-spec.js`);
+  });
+
+  it(`should return a generic spec glob if neither 'desktop' nor 'mobile' is specified`, () => {
+    const config = makeMockConfig();
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/**/*-spec.js`);
+  });
+
+  it(`should return a desktop spec glob if only 'desktop' is specified`, () => {
+    const config = makeMockConfig({desktop: true});
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/{desktop/**/,,!(mobile)/**/}*-spec.js`);
+  });
+
+  it(`should return a mobile spec glob if only 'mobile' is specified`, () => {
+    const config = makeMockConfig({mobile: true});
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/{mobile/**/,,!(desktop)/**/}*-spec.js`);
+  });
+
+  it(`should return a file-specific glob if a file is specified with no platforms`, () => {
+    const file = 'whatever-spec';
+    const config = makeMockConfig({file});
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/**/${file}.js`);
+  });
+
+  it(`should return a file-specific glob if a file is specified with desktop`, () => {
+    const file = 'whatever-spec';
+    const config = makeMockConfig({file, desktop: true});
+    expect(makeSpecsGlob(config, runnerOptions)).to.equal(`${baseSpecDir}/{desktop/**/,,!(mobile)/**/}${file}.js`);
+  });
+});
+
+
+describe(`#getTestFiles`, () => {
+  function compareMaps(mapA, mapB) {
+    expect(JSON.stringify(mapA.toJSON())).to.equal(JSON.stringify(mapB.toJSON()));
+  }
+
+  function getTests(...tests) {
+    return [].concat.apply(Array.prototype, tests).sort();
+  }
```

Comment:
fyi when you use spread, `...tests` will already be an Array. also, you would do `[].prototype.slice.call(arguments)`